docs/AppFramework.md

```diff
@@ -30,7 +30,8 @@ In this example, you'll deploy a Standalone CR with a remote storage volume, the
 
 Example: Standalone.yaml
 
-```apiVersion: enterprise.splunk.com/v1
+```
```

Comment:
Nit: Is there a reason we moved the apiVersion to a new line here?

Reply:
vscode preview was missing the apiVesion in preview, if it is in same line. So, I had to format this.
Actually, all these deserve YAML formatting, let me address it.